src/main/java/Duke.java

```diff
@@ -1,10 +1,181 @@
+
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
+
+        // Welcome Message
+        PrintWelcomeMessage();
+
+        // Active Chat
+        ActiveChat();
+
+        // Goodbye Message
+        PrintGoodbyeMessage();
+    }
+
+    private static void ActiveChat() {
```

Comment:
I think 'ActiveChat' should be 'activeChat' instead, since methods should be in camelCase.